### PR TITLE
Create VPN document

### PR DESCRIPTION
**Pull request by jpwalden**
Signed-off-by: jari-pekka walden <jari-pekka.walden@jolla.com>

### CODE REVIEW

File: Support/Help_Articles/VPN/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+2. Find heading/link "Free OpenVPN Account" and tap that.
+3. Tap one of the servers below, e.g., "PL226 OpenVPN Certificate Bundle". Choose location where to download the file, for example under "Downloads".
+Also, pay attention to the username and password at the end of the server list - you will need them later.
+4. Open Settings > Transfers. You will see "VPNbook.com-OpenVPN-PL226.zip" at the top of the list. Tap it.
+5. List of included ".ovpn" files appears. Pull down "Extract all". Those files are now ready to be used in "Downloads > VPNBook" folder.
+6. Open Settings > VPN. Pull down "Add new VPN".
+7. Tap "OpenVPN". Tap "Import file". Tap "Home folder".  Tap "Downloads". Tap "VPNBook".
+8. Select what kind of VPN service you want to use. Let's say it is TCP via port 80. Tap "vpnbook-pl226-tcp80".
+9. Tap "Accept" at the top right corner and Page "OpenVPN set up is ready" appears.
+10. Tap "vpnbook-de4-tcp80" to enable the VPN service.
```
Comment (jovirkku) [inline Support/Help_Articles/VPN/README.md:24]: Not "vpnbook-de4-tcp80" but "vpnbook-pl226-tcp80", right?

File: Support/Help_Articles/VPN/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+11. Type "vpnbook" for the Username. Tap the password from step 4 to the Password line. Tap "Remember credential information".
+12. Tap connect. You should see "Connected" under "vpnbook-pl226-tcp80".
+13. Go to the Home view and look at the status icons at the top of the display: there is the VPN icon "Shield" indicating that VPN is active.
+14. Star browsing over the VPN connection.
```
Comment (jovirkku) [inline Support/Help_Articles/VPN/README.md:28]: Star**t**

**Review by jovirkku — APPROVED** [review]

File: Support/Help_Articles/VPN/README.md (hunk 1 above)
Comment (jpwalden) [reply Support/Help_Articles/VPN/README.md]: Yes, missed that one when updated the others, fixed

File: Support/Help_Articles/VPN/README.md (hunk 2 above)
Comment (jpwalden) [reply Support/Help_Articles/VPN/README.md]: thanks -fixed

**Review by vigejolla — APPROVED** [review]

**Pull request merged**